internal/http/services/owncloud/ocdav/get.go

```diff
@@ -134,3 +138,8 @@ func (s *svc) handleGet(w http.ResponseWriter, r *http.Request, ns string) {
 		log.Error().Err(err).Msg("error finishing copying data to response")
 	}
 }
+
+// Rawurlencode https://www.php2golang.com/method/function.rawurlencode.html
+func Rawurlencode(str string) string {
```

Comment:
Why the url.QueryEspace is not enough?

Reply:
oc10 encodes filename in `Content-Disposition` header with PHP's `rawurlencode` function. The difference between `rawurlencode` and `url.QueryEscape` is `url.QueryEscape` encodes spaces as `+` instead of `%20`. I added this function to ensure consistency with oc10.

I found `url.PathEscape` is also working in same way. So, now `url.PathEscape` has been used instead of this function with the last commit.

Reply:
@karakayasemi in that case, as youre' doing path.Base there no need to call `url.PathEscape` as there isn't a path. Base gives you the basename of the path.

Reply:
Yes, you are right. Redundant `url.PathEscape` has been removed.